### PR TITLE
refactor: capitalize immutable variables 

### PR DESCRIPTION
**Pull request by andreivladbrg**
Closes https://github.com/sablier-labs/v2-core/issues/699

Note: being a small change I think this shouldn't be included in the `CHANGELOG` .

### CODE REVIEW

File: test/fork/Fork.t.sol [hunk 1]
```diff
@@ -12,8 +12,8 @@ abstract contract Fork_Test is Base_Test {
                                      CONSTANTS
     //////////////////////////////////////////////////////////////////////////*/
 
-    IERC20 internal immutable asset;
-    address internal immutable holder;
+    IERC20 internal asset;
+    address internal holder;
```
Comment (PaulRBerg) [inline test/fork/Fork.t.sol:16]: Why have you removed `immutable` here? Let's make them uppercase

**Review by PaulRBerg — CHANGES_REQUESTED** [review]
I agree that we should not include this in the changelog. This PR has nothing do with the public API.

File: test/fork/Fork.t.sol (hunk 1 above)
Comment (andreivladbrg) [reply test/fork/Fork.t.sol:16]: It was easier to remove the state, it is not relevant here to use `immutable` but as you wish.

Comment (PaulRBerg) [reply test/fork/Fork.t.sol]: It is relevant to have these variables remain immutable - the goal is to prevent accidental modifications.

**Review by PaulRBerg — APPROVED** [review]
now it looks good

**Pull request merged**